include/functions.h

```diff
@@ -5,15 +5,21 @@
 #include "macros.h"
 
 f32 fabsf(f32 f);
-#ifndef __sgi
-#define fabsf(f) __builtin_fabsf((f32)(f))
-#else
+#ifdef __sgi
 #pragma intrinsic(fabsf)
+#else
+#define fabsf(f) __builtin_fabsf((f32)(f))
 #endif
+
 f32 sqrtf(f32 f);
+#ifdef __sgi
 #pragma intrinsic(sqrtf)
+#endif
```

Comment:
Why the ifdefs for the pragmas? It shouldn't have any effect on other compilers

Reply:
Some compilers (gcc) will warn about unknown pragmas